### PR TITLE
Add 44444 to valid-servers

### PR DESCRIPTION
**Pull request by lucasgautheron**

### CODE REVIEW

**Comment by lucasgautheron** [comment]
No idea why it fails to build now ?

File: accumulation/src/main/resources/af/acc/valid-servers.json [hunk 1]
```diff
@@ -79,6 +79,26 @@
...
+    "name": "se 1111",
+    "timezone": "America/New_York"
+  },
+  "153.92.126.156 weed-lounge AssaultCube[local#2222]": {
+    "name": "se 2222",
+    "timezone": "America/New_York"
+  },
+  "153.92.126.156 weed-lounge AssaultCube[local#3333]": {
+    "name": "se 3333",
+    "timezone": "America/New_York"
```
Comment (ScalaWilliam) [inline accumulation/src/main/resources/af/acc/valid-servers.json]: SE 3333 == new york? I am so confus

**Comment by ScalaWilliam** [comment]
Build server too slow to run that test properly. I'll add a longer timeout

File: accumulation/src/main/resources/af/acc/valid-servers.json (hunk 1 above)
Comment (lucasgautheron) [inline accumulation/src/main/resources/af/acc/valid-servers.json]: yep....

**Pull request merged**